src/app/leaves/components/LeaveActionRender.tsx

```diff
@@ -76,15 +112,33 @@ const LeaveTableActionRender = ({ leaveRequestDetails }: Props) => {
   const statusActions = useMemo(
     () => [
       {
-        title: "Approve",
+        title: "Accept",
         id: 4,
-        onClick: handleApprovedRequest,
+        onClick: () => {
+          if (requestId !== undefined) {
+            handleLeaveRequestStatus(requestId, LeavesRequestStatus.Approved);
+          } else {
+            toast({
+              title: "Error",
+              description: "Request ID is missing.",
+            });
+          }
+        },
         icon: <CheckCircleIcon size={16} />,
       },
       {
         title: "Reject",
```

Comment:
enum

Reply:
same here we don't have an enum for titles